ixwebsocket/IXSocketConnect.cpp

```diff
@@ -24,6 +25,10 @@
 
 namespace ix
 {
+    void logger(__attribute__((unused)) int level, const char* message, __attribute__((unused)) void* userdata)
```

Comment:
better prefer `[[maybe unused]]`, as it's a part of the C++ standard, and __attribute__ is a GCC extension.

Or even simpler way: use just a type (`int`) without argument name, and the compiler will not complain.